nbextensions/usability/toc2/main.js

```diff
@@ -307,7 +333,14 @@ var table_of_contents = function () {
           ul= ul.parent();
           while(!ul.is('ul')){ ul= ul.parent(); }
       }
-
+      // Change link id -- append current num_str so as to get a kind of unique anchor 
```

Comment:
If this means (just came from the dentist and JS is not my strongest language...) that all headline IDs get changed then this means that all links get bad. A not so drastic solution would be to use additional IDs for the Toc links *if these links are not unique*.